projects/bp-gallery/src/components/common/OverviewContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconProps, Tab, Tabs } from '@mui/material';
+import { Children, PropsWithChildren, ReactElement, useState } from 'react';
+
+const OverviewContainer = ({
+  titles,
+  icons,
+  children,
+}: PropsWithChildren<{ titles?: string[]; icons: ReactElement<IconProps>[] }>) => {
+  const childrenArray = Children.toArray(children);
```

Comment:
Apparently using Children is uncommon and can lead to fragile Code: [source](https://react.dev/reference/react/Children)